catalyst/contrib/models/segmentation/encoder/resnet.py

```diff
@@ -37,12 +37,33 @@ def __init__(
         arch: str = "resnet18",
         pretrained: bool = True,
         requires_grad: bool = None,
-        layers_indices: List[int] = None
+        layers_indices: List[int] = None,
+        state_dict: dict = None,
```

Comment:
what do you think about `Path` support for this arg?
It would be easier to use from Config API.

Reply:
ok :)